R/subject_industry.R

```diff
@@ -1047,11 +1067,22 @@ crosstab_text <- function(tables_data_grouped, subjectinput, YAGinput, countinpu
         "BTEC", "Other", "Not known"
       )
 
-      crosstab_text <- paste("For first degree graduates of ", subjecttext, ", ", YAGtext, " after graduation, the prior attainment band
+      if(is.na(first(topindustry_highest$prop)) == FALSE){
+      if (first(grad_numbers$grad_numbers, order_by = -grad_numbers$grad_numbers) > 0) {
+        prior_attainment_text <- paste0("For first degree graduates of ", subjecttext, ", ", YAGtext, " after graduation, the prior attainment band
                            with the highest number of graduates was `", first(grad_numbers$prior_attainment, order_by = -grad_numbers$grad_numbers), "`.
-                           Within this prior attainment band, the most common industry was <b>",
-        first(topindustry$SECTIONNAME, order_by = -topindustry[2]), "</b>, and the median earnings for graduates with this prior
-                       attainment band working in this industry were <b>£", format(max(crosstabs_earnings_data2), big.mark = ",", scientific = FALSE),
+                           Within this prior attainment band, ",pluraltext,
+                                       format_filtervalues(topindustry_highest$SECTIONNAME), ", and the median earnings for graduates with this prior
```

Comment:
A very small point... just wondering if rather than "median earnings for graduates _with_ this prior attainment band" should be "median earnings for graduates _in_ this prior attainment band" or "median earnings for graduates _within_ this prior attainment band".
Just a thought as I was reading through the text on the dashboard. Happy to just leave as is though, if you think it reads ok :)